flow/record/adapter/elastic.py

```diff
@@ -58,6 +78,14 @@ def record_to_document(self, record: Record, index: str) -> dict:
             "_index": index,
             "_source": self.json_packer.pack(rdict),
         }
+
+        # Check if hash_record is set and hash record to md5 if flag is set.
+        # Note: the _record_metadata contains timestamp field of target-query output.
+        # An option to change this is to call the `self.json_packer.pack` again on the
+        # `record._asdict()` row.
```

Comment:
What do you mean with this timestamp comment?

Reply:
Nevermind. Removed for now. Maybe later a code change for dealing a different way with the `_generated` timestamp field.